src/Process/PointCloudProcessing.js

```diff
@@ -86,6 +86,15 @@ function markForDeletion(elt) {
     }
 }
 
+function updateMinMaxDistance(context, bbox) {
```

Comment:
this isn't a calculation equivalent to those performed in 3Dtiles?

Reply:
it should be factorized then or use the [aggregation bboxs](https://github.com/iTowns/itowns/pull/810#pullrequestreview-139481142) to make the generic method

Reply:
Done.